### PR TITLE
Update GOVERNANCE.md on election details

### LINKED ISSUE #164: Update GOVERNANCE.md to state the process for election eligibility exceptions
### Description
GOVERNANCE.md states that maintainers are eligible for the TC but doesn't document the process for handling exceptions to this rule. This is however covered by the [TC elections document](https://hackmd.io/@eiffel-community/H1Oyst_4u). Let's make that policy more explicit and official by moving it into the charter.

### Motivation
The charter should include all applicable rules and elections processes and the TC elections document should concentrate on the execution of said process.

### Exemplification
N/A

### Benefits
All rules in one place. Right now you have to cross-reference both documents to understand how a TC election should be carried out.

### Possible Drawbacks
None.

### PR DESCRIPTION
**Pull request by e-backmark-ericsson**
### Applicable Issues
Closes #164 

### Description of the Change
Updates to the GOVERNANCE.md document to clarify the TC election process, including the review of the eligible candidates list. This PR goes hand-in-hand with the updates to the ELECTIONS.md document seen [here](https://github.com/eiffel-community/community/commit/63b4e9763ca180b99c1335e67dafeb644f4d680c) and [here](https://github.com/eiffel-community/community/commit/824a2cc97752b262e82979bfa5fb260378182662) (updated through HackMD).

### Alternate Designs
N/A

### Benefits
Clearer election process, and no duplication 

### Possible Drawbacks
none

### Sign-off
Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Emil Bäckmark emil.backmark@ericsson.com

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -150,31 +150,30 @@ Information related to conducted and ongoing elections are kept in [ELECTIONS.md
...
 
-Anyone who is a maintainer of one or more projects hosted by Eiffel Community is eligible to run for a seat in Eiffel Technical Committee or nominate another maintainer as a candidate.
-
-Anyone who is a member to [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community) is eligible to vote on Technical Committee Elections.
-
-Community elections are overseen by [Election Officers](#Election-Officers).
-
-The members of Technical Committee are elected using [Ranked Voting](https://en.wikipedia.org/wiki/Ranked_voting).
+1. Anyone who is a maintainer of one or more projects hosted in the [Eiffel Community GitHub organization](https://github.com/eiffel-community) is eligible to run for a seat in the Eiffel Technical Committee or nominate another maintainer as a candidate. The list of eligible candidates will be sent for review to the Eiffel Community prior to the start of the nomination phase.
+1. Anyone who is a member to [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community) is eligible to vote on Technical Committee Elections.
```
Comment (magnusbaeck) [inline GOVERNANCE.md:154]:
```suggestion
1. Anyone who is a member of the [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community) is eligible to vote on Technical Committee Elections.
```

File: GOVERNANCE.md [hunk 2]
```diff
@@ -150,31 +150,30 @@ Information related to conducted and ongoing elections are kept in [ELECTIONS.md
...
 
 #### Election Process
 
 The election process consists of 3 phases
 
 * **Preparation Phase**:
-  * Election Officers determine the list of community members who are eligible to **run** in community elections announce this on [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community).
+  * Election Officers prepare the timeline for the elections and get it approved by the Technical Committee.
+  * Election Officers determine the list of community members who are eligible to **run** in the Technical Committee elections and announce this on [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community).
+  * The list of eligible members is open for review for one to two weeks. Any changes to this list are managed by the Election Officers and informed to the Technical Committee before the nomination phase starts.
```
Comment (magnusbaeck) [inline GOVERNANCE.md:166]:
```suggestion
  * The list of eligible members is open for review for one to two weeks. Any changes to this list are managed by the Election Officers who inform the Technical Committee before the nomination phase starts.
```

**Review by magnusbaeck — CHANGES_REQUESTED** [review]
Apart from the language nits, the linked issue talks about the eligibility exception process but I can't see any changes related to that in this PR.

**Comment by e-backmark-ericsson** [comment]
I couldn't find any explicit exemption process mentioned in the elections document, so I thought the only thing was that the list of eligible candidates was sent for review to the community and then updated if any candidates are missing or faultily added there. Are there any other exemptions we should document?

**Comment by e-backmark-ericsson** [comment]
Update the text on the eligible candidate list slightly. Please re-review.

**Review by magnusbaeck — CHANGES_REQUESTED** [review]
Found another language nit that we might as well fix, otherwise LGTM.

**Review by magnusbaeck — APPROVED** [review]

**Review by m-linner-ericsson — APPROVED** [review]

**Pull request merged**